Docker/min.Dockerfile

```diff
@@ -0,0 +1,76 @@
+FROM ubuntu:18.04
+
+#Setup Ansible
+ENV ANSIBLE_VERSION 2.9.4
+ENV DEBIAN_FRONTEND noninteractive
+
+COPY ansible /tmp/ansible
+
+ENV TZ=UTC-4
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+RUN set -x && \
+    echo ">> Adding build-dependencies..." && \
+    sed -i 's/# \(.*multiverse$\)/\1/g' /etc/apt/sources.list && \
+    apt-get update && \
+    apt-get -y upgrade && \
+    apt-get install -y build-essential \
+    software-properties-common \
+    apt-transport-https \
+    git \
+    wget \
+    && apt-add-repository ppa:ansible/ansible && \
+    apt-get update && \
+    apt-get install -y ansible && \
+    echo ttf-mscorefonts-installer msttcorefonts/accepted-mscorefonts-eula select true | debconf-set-selections
+
+RUN ansible-playbook -i "localhost," -c local /tmp/ansible/devbox.yml
+
+ENV ANSIBLE_GATHERING smart
+ENV ANSIBLE_HOST_KEY_CHECKING false
+ENV ANSIBLE_RETRY_FILES_ENABLED false
+ENV ANSIBLE_ROLES_PATH /ansible/playbooks/roles
+ENV ANSIBLE_SSH_PIPELINING True
+ENV PYTHONPATH /ansible/lib
+ENV PATH /ansible/bin:$PATH
+ENV ANSIBLE_LIBRARY /ansible/library
+
+WORKDIR /root
+
+ENTRYPOINT ["/bin/bash"]
+
+# Install OpenCV 3.2.0
+RUN apt-get update -y && \
+    apt-get upgrade -y && \
+    apt-get autoremove -y && \
+    apt-get install -y --no-install-recommends cmake git libgtk2.0-dev pkg-config libavcodec-dev libavformat-dev \ 
+    libswscale-dev python3.6-dev python3-numpy libtbb2 libtbb-dev libjpeg-dev libpng-dev libtiff5-dev jasper \
+    libdc1394-22-dev libeigen3-dev libtheora-dev libvorbis-dev libxvidcore-dev libx264-dev sphinx-common libtbb-dev \
+    yasm libfaac-dev libopencore-amrnb-dev libopencore-amrwb-dev libopenexr-dev libgstreamer-plugins-base1.0-dev \
+    libavutil-dev libavfilter-dev libavresample-dev
+
+RUN apt-get install -y gcc
+
+RUN cd /opt && \
+    wget --no-check-certificate -O opencv-3.2.0.tar.gz https://github.com/opencv/opencv/archive/3.2.0.tar.gz && \
+    tar -xzf opencv-3.2.0.tar.gz && \
+    wget --no-check-certificate -O opencv_contrib-3.2.0.tar.gz https://github.com/opencv/opencv_contrib/archive/3.2.0.tar.gz && \
+    tar -xzf opencv_contrib-3.2.0.tar.gz && \
+    rm opencv-3.2.0.tar.gz && \
+    rm opencv_contrib-3.2.0.tar.gz && \
+    cd opencv-3.2.0 && \
+    mkdir release && \
+    cd release && \
+    cmake -D WITH_CUDA=OFF -D CMAKE_BUILD_TYPE=RELEASE -D CMAKE_INSTALL_PREFIX=/usr/local \
```

Comment:
is this the only way to do this? Not really an issue but seems quite verbose. I thought by default it won't build optional modules unless you specify them. Theres also some CMake config file I think.